README.md

```diff
@@ -15,7 +16,7 @@ docker-compose up -d
 ## Components
 
 This Docker Compose includes the following components:
-* [Abacus Inferno Fork](https://git.codev.mitre.org/projects/ABA/repos/cypress-inferno/browse)
+* [Abacus Inferno Fork](https://github.com/projecttacoma/deqm-test-client)
 * [CQF Ruler](https://hub.docker.com/u/contentgroup)
 * [NGINX](https://hub.docker.com/_/nginx) NDJSON File Server
```

Comment:
NGINX, Keycloak, and Bellese Bulk Data Proxy were removed from the default `docker-compose.yml` in https://github.com/projecttacoma/docker-deqm-test-client/pull/4 just now, so we could separate those into a "Development Components" section or something along those lines.